include/openssl/kdf.h

```diff
@@ -29,6 +29,9 @@ extern "C" {
 # define EVP_PKEY_CTRL_SCRYPT_R                 (EVP_PKEY_ALG_CTRL + 11)
 # define EVP_PKEY_CTRL_SCRYPT_P                 (EVP_PKEY_ALG_CTRL + 12)
 # define EVP_PKEY_CTRL_SCRYPT_MAXMEM_BYTES      (EVP_PKEY_ALG_CTRL + 13)
+# define EVP_PKEY_CTRL_PBKDF2_MD                (EVP_PKEY_ALG_CTRL + 14)
+# define EVP_PKEY_CTRL_PBKDF2_SALT              (EVP_PKEY_ALG_CTRL + 15)
```

Comment:
I would rather rename `EVP_PKEY_CTRL_SCRYPT_SALT` to `EVP_PKEY_CTRL_SALT` and reuse that.